scarlet/display.py

```diff
@@ -225,7 +225,7 @@ def show_scene(
     panels = sum((show_model, show_observed, show_rendered, show_residual))
     if linear:
         if figsize is None:
-            figsize = (3 * panels, 3 * len(list(sources)))
+            figsize = (5 * panels, 5)
```

Comment:
I'm not sure the larger figures will work as well in the docs, but let's find out